plugins/wasm-go/extensions/request-block/main.go

```diff
@@ -104,7 +119,13 @@ func onHttpRequestHeaders(ctx wrapper.HttpContext, config RequestBlockConfig, lo
 			requestUrl = strings.ToLower(requestUrl)
 		}
 		for _, blockUrl := range config.blockUrls {
-			if strings.Contains(requestUrl, blockUrl) {
+			if requestUrl == blockUrl {
```

Comment:
Why change this? This change will make this plugin incompatible with previous version.

Reply:
This may require discussing whether using regular matching is more reasonable in the form of substring matching. I feel that retaining two matching modes (precise matching and regular matching) is more reasonable

Reply:
I think we should talk about the new configuration format while maintaining configuration compatibility. I suggest you design an example of the new configuration format, and we will discuss it based on this.

Reply:
Reserve block-url for string inclusion matching, add new blocks_ Exact_ URLs for precise URL matching